packages/web3-eth/types/index.d.ts

```diff
@@ -145,6 +145,8 @@ export class Eth extends AbstractWeb3Module {
     submitWork(data: [string, string, string], callback?: (error: Error, result: boolean) => void): Promise<boolean>;
 
     pendingTransactions(callback?: (error: Error, result: []) => void): Promise<[]>;
+
+    getProof(address: string, storageKey: string[], blockNumber: number | string, callback?: (error: Error, result: GetProof) => void): Promise<GetProof>;
```

Comment:
```blockNumber: number | string | "latest" | "earliest"``` will make this even more clearer to people.

Reply:
Ya think so.